releases/stellar/2025-01-ITS-v1.0.0.md

```diff
@@ -88,71 +149,71 @@ WALLET=wallet_address
 ```bash
 node stellar/its.js deploy-interchain-token TEST test 18 0x1234 10000
 
-node stellar/its.js deploy-remote-interchain-token 0x1234 avalanche
+node stellar/its.js deploy-remote-interchain-token 0x1234 [destination-chain]
 
-node evm/gateway.js -n avalanche --action execute --payload $PAYLOAD --sourceChain axelar --sourceAddress $ITS_HUB_ADDRESS --messageId [message-id] --destination $WALLET
+node evm/gateway.js -n [destination-chain] --action execute --payload $PAYLOAD --sourceChain axelar --sourceAddress $ITS_HUB_ADDRESS --messageId [message-id] --destination $WALLET
 ```
 
 2. Interchain Token Transfer for Native Interchain Token
 
 ```bash
-node stellar/its.js interchain-transfer 0xa98974cd02e5ee3d65bff69465a00917f27176bffd72e352c11a78c7a999bded avalanche $WALLET 1
+node stellar/its.js interchain-transfer 0xa98974cd02e5ee3d65bff69465a00917f27176bffd72e352c11a78c7a999bded [destination-chain] $WALLET 1
 
-node evm/gateway.js -n avalanche --action execute --payload $PAYLOAD --sourceChain axelar --sourceAddress $ITS_HUB_ADDRESS --messageId [message-id] --destination $WALLET
+node evm/gateway.js -n [destination-chain] --action execute --payload $PAYLOAD --sourceChain axelar --sourceAddress $ITS_HUB_ADDRESS --messageId [message-id] --destination $WALLET
```

Comment:
This cmd is a generic execute cmd. You can mention it once initially instead of duplicating